protoc-gen-kotlin/lib/src/commonMain/kotlin/pbandk/gen/File.kt

```diff
@@ -23,7 +26,17 @@ data class File(
             ret += protobufTypeName to kotlinTypeName
             (t as? Type.Message)?.nestedTypes?.forEach { applyType(it, protobufTypeName, kotlinTypeName) }
         }
+
+        fun applyExtension(f: Field) {
+            if (f is Field.Numbered) {
+                if (!ret.containsKey(f.extendee!!)) {
+                    ret += f.extendee!! to f.extendee!!.trimStart('.').replace("google.protobuf", "pbandk.wkt")
```

Comment:
This was added for specifying the package + type for extensions as `kotlinQualifiedTypeName` only has the type available and `f.extendee` would work by itself but it has a leading `.`.  This definitely could me just not understanding this correctly. We talked about this on the phone before and i thought we agreed this was what we wanted but again i could just be confused.
```
package pbandk.testpb;


extend google.protobuf.FieldOptions {
    // Rules specify the validations to be performed on this field. By default,
    // no validation is performed against a field.
    optional FieldRules rules = 1071;
}
```
Would try to generate the code with the incorrect package as `pbandk.testpb.FieldOptions` instead of the `pbandk.wkt.FieldOptions` 

we want the above to produce

```val pbandk.wkt.FieldOptions.rules: pbandk.testpb.FieldRules? get() =
    getExtension(pbandk.testpb.rules)

val rules = pbandk.FieldDescriptor(
    messageDescriptor = pbandk.wkt.FieldOptions.Companion::descriptor,
    name = "rules",
    number = 1071,
    type = pbandk.FieldDescriptor.Type.Message(messageCompanion = pbandk.testpb.FieldRules.Companion),
    jsonName = "rules",
    value = pbandk.wkt.FieldOptions::rules
)
```

Reply:
this was cleared up below, thanks for being so thorough